### ISSUE TITLE
Support for `-config`?

### ISSUE THREAD

**Issue by trash-panda** [janfri/multi_exiftool#13]
Hello!

I have an `.ExifTool_config` file that defines some custom tags.  To use it, the exiftool CLI has the `-config` option, which  [must come before all other arguments on the command line][0].  However, when I try to do this with MultiExiftool:

```ruby
      writer = MultiExiftool::Writer.new
      writer.options = { 'config' => '/path/to/.ExifTool_config', 'P' => true, 'E' => true }
      writer.filenames = file_path
      writer.overwrite_original = true
      writer.values = values
```

The `-config` option is always preceded by some `-charset` options:

```pry > writer.exiftool_args                                                                           
=> ["-charset",                                                                                                                        
  "FileName=utf8",                                                                                                                      
  "-charset",                                                                                                                         
  "utf8",                                                                                                                            
  "-config",                                                                                                                        
  "/path/to/.ExifTool_config",
  "-P",                                                                                                                               
  "-E",                                                                                                                               
  "-overwrite_original",
  ...
```

Is there some way to get the `.exiftool_args` to start with `-config?`

[0]: https://www.exiftool.org/exiftool_pod.html#config-CFGFILE

**Comment by janfri** [comment]
I will have a look.

**Comment by janfri** [comment]
I've just released multi_exiftool 0.16.0.

Now you can do the following:
```ruby
      writer = MultiExiftool::Writer.new
      writer.config = '/path/to/.ExifTool_config'
      writer.options = {'P' => true, 'E' => true }
      writer.filenames = file_path
      writer.overwrite_original = true
      writer.values = values
```
or alternatively with the functional api:
```ruby
  values, errors = MultiExiftool.write(file_path, values, config: '/path/to/.ExifTool_config', 'P' => true, 'E' => true, overwrite_original: true)
```
Let me know if it works for you.

**Comment by trash-panda** [comment]
It works great, thank you!
